### PR TITLE
Updated the Travis CI config to allow failures for Vale and Netlify stages

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Added `allow_failures` to `check-with-vale` and `netlify` stages:
Based on the solution at https://github.com/travis-ci/travis-ci/issues/8998#issuecomment-657095023 we want allow failures for these stages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,11 +54,14 @@ jobs:
...
         - python3 build.py --distro microshift --product "MicroShift" --version 4 --no-upstream-fetch && python3 makeBuild.py
     - stage: check-with-vale
+      env:
+        - CAN_FAIL=true
       if: type IN (pull_request)
       name: "Run Vale against PR asciidoc files"
       before_script:
         - gem install asciidoctor
       script:
+        - echo "EXITING stage" && exit 1
```
Comment (gaurav-nelson) [inline .travis.yml:64]: This line is for testing only, we should remove this before we merge.

File: .travis.yml [hunk 2]
```diff
@@ -66,9 +73,12 @@ jobs:
     #   if: env(PR_AUTHOR)=openshift-cherrypick-robot
     #   script: bash ./automerge.sh
     - stage: netlify
+      env:
+        - CAN_FAIL=true
       language: minimal
       if: branch IN (main, enterprise-4.11, enterprise-4.12)
       script:
+        - echo "EXITING stage" && exit 1
```
Comment (gaurav-nelson) [inline .travis.yml:81]: This line is for testing only, we should remove this before we merge.

**Review by aireilly — COMMENTED** [review]
LGTM

**Pull request merged**